tests/runtime/call

```diff
@@ -367,7 +367,7 @@ TIMEOUT 1
 NAME path_in_unsupported_kfunc
 PROG kfunc:vfs_read { print(path(args->file->f_path)); }
 EXPECT \nstdin:1:18-48: ERROR: helper bpf_d_path not supported in probe\n
-REQUIRES_FEATURE dpath
+REQUIRES_FEATURE dpath kfunc
```

Comment:
We prefer this style:
```suggestion
REQUIRES_FEATURE dpath
REQUIRES_FEATURE kfunc
```
It makes it easier to spot that the test depends on multiple features.

Reply:
Ack.